css/main.css

```diff
@@ -471,7 +470,25 @@ table th, table tr , table td{
   .halls-radio-container {
     display: block;
   }
-
+  .subject-select-container
+  {
+    display: block;
+    width: 23em;
+    overflow: hidden;
+    margin: 0 1.5em;
+    border: 0.1em solid;
+    border-radius: 0.25em;
+  }
+  .subjectselect
+  {
+     border:none;
+     overflow: hidden;
+     cursor: pointer;
+     text-overflow: ellipsis;
+     white-space: nowrap;
+     width: 18.75em;
```

Comment:
The options look a little odd. Better to avoid fixing width property. Do you wish to take a look at how [Bootstrap](http://getbootstrap.com/) implement select options ? Else, i'll take care of this.
![crop](https://cloud.githubusercontent.com/assets/12448084/25385199/dee1c4a0-29e0-11e7-8c81-b8487cad3f90.png)